uint32 seq
---
geometry_msgs/PoseArray pose_list